### PR TITLE
redesign block/homepage

### PR DESCRIPTION
**Pull request by chovy**
- add sidebar navigation
- dark color scheme
- change font colors and link colors
- add rounded corners to input fields
- use svg logo

add margin to page wrapper

- account for new navbar on left

add colors to tx and address pages

### CODE REVIEW

**Comment by chovy** [comment]
screenshot 2018-05-21 15 31 13

File: .env.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export PORT=8000
+export MONGO_URI=mongodb://localhost:27017/blockDB
+export RPC_URL=https://testnet-rpc.gochain.io:443
```
Comment (rkononov) [inline .env.sh]: I think you shouldn't post real values here

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    image: gochain/explorer:latest
+    command: "node app.js"
+    links:
+        - "mongodb"
+    ports:
+        - "80:80"
+    depends_on:
+      - mongodb
+    environment:
+          - RPC_URL=https://testnet-rpc.gochain.io:443
```
Comment (rkononov) [inline docker-compose.yml]: same here use env variables

File: .env.sh (hunk 1 above)
Comment (rkononov) [reply .env.sh]: or just remove this file from commit

File: docker-compose.yml (hunk 2 above)
Comment (rkononov) [reply docker-compose.yml]: or just remove this file from commit

**Pull request merged**